src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/DockerBomTool.groovy

```diff
@@ -62,8 +62,8 @@ class DockerBomTool extends BomTool {
         if (!propertiesOk) {
             logger.debug('The docker properties are not sufficient to run')
         } else {
-            dockerExecutablePath = executableManager.getPathOfExecutable(ExecutableType.DOCKER, detectConfiguration.dockerPath)?.trim()
-            bashExecutablePath = executableManager.getPathOfExecutable(ExecutableType.BASH, detectConfiguration.bashPath)?.trim()
+            dockerExecutablePath = getExecutablePath(ExecutableType.DOCKER, true, detectConfiguration.dockerPath)
```

Comment:
Is this supposed to be `findExecutablePath(....)` not `getExecutablePath(...)` ?